core/src/main/scripts/importer/allowed_data_types.txt

```diff
@@ -44,5 +44,6 @@ PROTEIN_LEVEL	CONTINUOUS	protein_quantification
 PROTEIN_LEVEL	Z-SCORE	protein_quantification_zscores
 GENESET_SCORE	GSVA-SCORE	gsva_scores
 GENESET_SCORE	P-VALUE	gsva_pvalues
+TREATMENT	LIMIT-VALUE	*
```

Comment:
Here I also would like to see the reference to the TREATMENT (a.k.a. TREATMENT_RESPONSE) be removed.